base/src/Module.cpp

```diff
@@ -1295,6 +1298,16 @@ bool Module::step()
     {
       throw AIPException(CTRL_MODULE_INVALID_STATE, "Unexpected: " + std::to_string(frames.size()) + " frames remain unprocessed in control module.");
     }
+    if (mPlay)
+    {
+        mProfiler->startProcessingLap();
+        ret = stepNonSource(frames);
```

Comment:
Why we need if(mPlay) block in term call

Reply:
not in term